android-components/plugins/dependencies/src/main/java/DependenciesPlugin.kt

```diff
@@ -49,7 +49,7 @@ object Versions {
 
     // see https://android-developers.googleblog.com/2022/06/independent-versioning-of-Jetpack-Compose-libraries.html
     // for Jetpack Compose libraries versioning
-    const val compose_version = "1.2.1"
+    const val compose_version = "1.3.0"
```

Comment:
@mcarare Did you mean to change this to 1.3.0?

Reply:
No, I tested it with both 1.3.0 and 1.3.1 and committed the old value. Corrected it, thank you for the heads-up!